django/db/models/functions/base.py

```diff
@@ -205,3 +205,18 @@ def as_oracle(self, compiler, connection):
 class Upper(Transform):
     function = 'UPPER'
     lookup_name = 'upper'
+
+
+class Ltrim(Transform):
+    function = 'LTRIM'
+    lookup_name = 'ltrim'
+
+
+class Rtrim(Transform):
+    function = 'RTRIM'
+    lookup_name = 'rtrim'
+
+
+class Trim(Transform):
```

Comment:
SQLite, Oracle, PostgreSQL, MySQL and just to put in the list MSSQL support `trim(X, Y)`. The syntax varies a bit, and the function can be used to trim from either end, or both on both PostgreSQL and MySQL. By default on the other backends, it's both ends. Maybe supporting it both as a transform for whitespace, and as a function to strip away characters is a common usage scenario?

Reply:
@atombrella Thanks for suggestion. I added `characters` to all functions.